cirq/circuits/circuit.py

```diff
@@ -1210,14 +1206,7 @@ def _resolve_operations(
         ext: extension.Extensions) -> List[ops.Operation]:
     resolved_operations = []  # type: List[ops.Operation]
     for op in operations:
-        cast_op = ext.try_cast(ops.ParameterizableEffect, op)
-        if cast_op is None:
-            resolved_op = op
-        else:
-            resolved_op = cast(
-                    ops.Operation,
-                    cast_op.with_parameters_resolved_by(param_resolver))
-        resolved_operations.append(resolved_op)
+        resolved_operations.append(cirq.resolve_parameters(op, param_resolver))
```

Comment:
Does this make `ext` a dead parameter? If so, delete it (and update calling methods to not pass it in, and delete ext from those methods if it becomes dead for them also)